### PR TITLE
Add flow logs for all failed vpc connections

### PR DESCRIPTION
**Pull request by deanwilson**

### CODE REVIEW

File: terraform/projects/infra-flow-logs/main.tf [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        "Service": "vpc-flow-logs.amazonaws.com"
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy" "vpc_flow_logs_policy" {
```
Comment (SamLR) [inline terraform/projects/infra-flow-logs/main.tf]: Can this use a [policy doc data resource](https://www.terraform.io/docs/providers/aws/d/iam_policy_document.html) so there's some amount of verification before it hits AWS?

**Comment by afda16** [comment]
Does it have to be a different project? We could put it with infra-vpc, with a variable to enable logs if selected

**Comment by afda16** [comment]
I updated the policy creation to use a json file, same as the rest of the code. I still think this code should be part of the infra-vpc project

**Comment by afda16** [comment]
I moved the code to `infra-vpc` and removed the `infra-flow-logs` project

**Pull request merged**